### DISCUSSION TITLE
q-input debounce has no effect on Android?

### DISCUSSION THREAD

**Discussion by elfxby** [quasarframework/quasar#13688] [CLI - Capacitor / Cordova modes] [answered]
The model-value of q-input is changed on Android only after pressing Enter on the virtual keyboard.
But I want to show the search results already when typing (with debounce).
How can I do that?

**Comment by pdanpdan** [comment, marked as answer]
https://github.com/quasarframework/quasar/pull/13658
https://github.com/quasarframework/quasar/pull/13659
